datasets/pubmed/pubmed.py

```diff
@@ -39,7 +39,7 @@
 # The HuggingFace dataset library don't host the datasets but only point to the original files
 # This can be an arbitrary nested dict/list of URLs (see below in `_split_generators` method)
 # Note these URLs here are used by MockDownloadManager.create_dummy_data_list
-_URLs = [f"ftp://ftp.ncbi.nlm.nih.gov/pubmed/baseline/pubmed22n{i:04d}.xml.gz" for i in range(1, 1115)]
+_URLs = [f"https://ftp.ncbi.nlm.nih.gov/pubmed/baseline/pubmed22n{i:04d}.xml.gz" for i in range(1, 1115)]
```

Comment:
Not sure why you changed the protocol. PubMed download instruction suggest using their FTP servers. 

Moreover, the change of protocol may have a performance impact.